### PR TITLE
added code snippet for counting frequency of characters in a string

### PR DESCRIPTION
**Pull request by yogeshsingh101200**
# Code snippet for counting the frequency of unique characters in a string

Added code snippet for counting frequency of unique characters in a string

Fixes # (issue)
#4 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generates no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by kaustubhgupta — APPROVED** [review]

File: snippets/python/count_frequency.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    for char in frequency:
+        file.write(f"Frequency of {char}: {frequency[char]}\n")
+```
+
+## Description
+Counter(a `dict` subclass) is a collection where elements are stored as dictionary keys and their counts are stored as dictionary values. [See docs](https://docs.python.org/3/library/collections.html#collections.Counter)
+
+## Output
+```
+Enter string: aabbc
```
Comment (sakshamarora1) [inline snippets/python/count_frequency.md:24]: I think adding this under input might be a better idea. and the next one shall remain under the output heading

File: snippets/python/count_frequency.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Counts frequency of unique characters in a string
```
Comment (sakshamarora1) [inline snippets/python/count_frequency.md]:
```suggestion
# Count frequency of unique characters in a string
```

File: snippets/python/count_frequency.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Frequency of a: 2
+Frequency of b: 2
+Frequency of c: 1
+```
+
+## Contributed By
+
+| Name | GitHub Username | Institute |
+| --- | --- | --- |
+| Yogesh Singh | yogeshsingh101200 | Maharaja Surajmal Intitute of Technology |
```
Comment (sakshamarora1) [inline snippets/python/count_frequency.md]: Great! Just a small typo.
```suggestion
| Yogesh Singh | yogeshsingh101200 | Maharaja Surajmal Institute of Technology |
```

Comment (yogeshsingh101200) [reply snippets/python/count_frequency.md]: @sakshamarora1 Oops! :smile:

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**